react/features/app-settings/components/AppSettings.native.js

```diff
@@ -80,20 +103,43 @@ class AppSettings extends AbstractAppSettings {
                             onValueChange = {
                                 this._onStartAudioMutedChange
                             }
-                            value = { this.state.startWithAudioMuted } />
+                            value = {
+                                this.props._profile.startWithAudioMuted
+                            } />
                     </FormRow>
                     <FormRow
                         i18nLabel = 'profileModal.startWithVideoMuted' >
                         <Switch
                             onValueChange = {
                                 this._onStartVideoMutedChange
                             }
-                            value = { this.state.startWithVideoMuted } />
+                            value = {
+                                this.props._profile.startWithVideoMuted
```

Comment:
general observation: I think it's worth it to deconstruct the props / profile to avoid long lines.